cluster/addons/limit-range/limit-range.json

```diff
@@ -0,0 +1,20 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "LimitRange",
+  "metadata": {
+    "name": "limits"
+  },
+  "spec": {
+    "limits": [
+    {
+      "type": "Container",
+      "min": {
+        "cpu": "400m"
```

Comment:
Why does the LimitRange need to be specified in multiple places (here and in salt)?

Why 2 different values? 

Why 400m? Do we think 100m won't be sufficient?

Do we really need a minimum? I'd prefer to just set a default, to make it easier to override in the case that someone actually specifies resources.